changelog/README.md

```diff
@@ -0,0 +1,15 @@
+The `changelog` directory contains the description of the changes introduced into the repository.
+The changes are essentially divided into 4 categories
+- changes: List of PRs brings Changes and/or Enhancements
+- bugfixes: List of PRs fixing existing issues
+- security: List of PRs fixing security issues
+- updates: Package Updates
+
+## How to add the file
+
+Based on the category the PR falls into create a new file in the respective directory
+with the filename format `DD-MM-YYYY-<few-words-about-the-change>.md`
+
+The contents of the file should describe the changes in an ellaborative manner, except for `Updates`.
```

Comment:
We could maybe add an entry example:

```md
$ cat `changelog/bugfixes/(date '+%d-%m-%Y')-selinux-torcx.md
add SELinux rule to allow torcx to correctly start in SELinux enforced mode ([#1426](https://github.com/flatcar-linux/coreos-overlay/pull/1426))
```
Since we give an `Update` example right after, that would make sense. :)

Reply:
Do you also have an example how to use flog to generate an update entry with a link to the changelog or an CVE list entry? Or is this not ready yet for this use case?

Reply:
```suggestion
The contents of the file should describe the changes in an elaborative manner, except for `Updates`.
```

Reply:
I did type the correct spelling of elaborative, not sure why vim suggested me double-l 😖

Reply:
The intention of `flog` is different, it would be a central place to all the project changelog. My idea is to have a script to do the release, something like [blurb](https://pypi.org/project/blurb/) but for us, it would be just a small script.